clients/web/src/lib/components/atoms/ProgressBar/ProgressBar.story.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import type {Hst as _Hst} from "@histoire/plugin-svelte";
+    import ProgressBar from "./ProgressBar.svelte";
+
+    export let Hst: _Hst;
+</script>
+
+<Hst.Story title="Atoms/ProgressBar">
+    <Hst.Variant title="Default">
+        <ProgressBar progress={75.6767676} />
+    </Hst.Variant>
+    <Hst.Variant title="With Label">
+        <ProgressBar label={true} size="large" />
+    </Hst.Variant>
+    <Hst.Variant title="With Label Inside">
+        <ProgressBar label={true} labelOutside={false} />
+    </Hst.Variant>
+    <Hst.Variant title="Show All Sizes">
+        <ProgressBar size="small" />
+        &nbsp;
+        <ProgressBar />
+        &nbsp;
+        <ProgressBar size="large" />
+        &nbsp;
+        <ProgressBar size="extraLarge" />
+    </Hst.Variant>
+</Hst.Story>
```

Comment:
If you're feeling brave, you can add controls to the story:
https://github.com/SprocketBot/sprocket/pull/388/files#diff-a85c60ed0cd92db705700a978da2082205340e579a6bd3aa4548b90ccffbeca8R20-R29

Not a huge deal either way

Reply:
I don't understand what this means. What does a svelte:fragment of slot="controls" do here?

Reply:
https://histoire.dev/guide/svelte3/controls.html

It adds controls in the right sidebar that allow you to tweak the props of the Component
![image](https://user-images.githubusercontent.com/31896377/201172161-89100706-29d6-42f6-bc7a-e8014f5ab0c3.png)

Reply:
`svelte:fragment` is an "invisible" element. Svelte can treat it like an Element, but it doesn't get rendered to the page. This lets us group children together (in this case, all the controls)

`slot="controls"` indicates to the parent (in this case `Hst.Story`), where it would like to be rendered. This only works if the parent has declared a slot of that name (i.e. `<slot name="controls"/>` somewhere in `Hst.Story`)

Reply:
Added in latest commit.